tests/integration_tests/test_llm.py

```diff
@@ -134,6 +140,20 @@ def test_llm_text_to_text(tmpdir, backend, ray_cluster_4cpu):
     assert preds["Answer_probabilities"]
     assert preds["Answer_probability"]
 
+    # Check that in-line generation parameters are used. Original prediction uses max_new_tokens = 5.
+    assert get_num_non_empty_tokens(preds["Answer_predictions"][0]) > 3
```

Comment:
One more way to guarantee this actually, is to use `min_new_tokens`: https://huggingface.co/docs/transformers/main/main_classes/text_generation#transformers.GenerationConfig.min_new_tokens in the base generation config, that way we are not left to determinism. Wdyt?

Reply:
Good idea! Done.